doc/user_guide/user_guide.md

```diff
@@ -1 +1,53 @@
-TDB
\ No newline at end of file
+# User Guide
+
+The Exasol Tableau Connector is distributed together with Tableau Desktop and Tableau Server applications. We recommend using the latest available version of Tableau products to access the connector.
+If you want to use the currently developed version of connector, you can follow the guide below and install the connector disabling sign verification.
+
+## Testing In-Development Connector
+
+## With Tableau Desktop
+
+* Clone the Tableau-Connector repository. You can execute the following command from [Git Bash](https://gitforwindows.org/) terminal:
+
+```
+git clone https://github.com/exasol/tableau-connector.git
+```
+
+* Open Windows cmd and start Tableau Desktop providing the path to the connector directory and also disabling the signature check.
+
+```
+"<path to tableau.exe file>" -DConnectPluginsPath=<path to connector> -DDisableVerifyConnectorPluginSignature
+```
+
+Example:
+
+```
+"C:\Program Files\Tableau\Tableau <version>\bin\tableau.exe" -DConnectPluginsPath=C:\Users\user\git\tableau-connector\src -DDisableVerifyConnectorPluginSignature
+```
+
+## With Tableau Server
+
+* Clone the Tableau-Connector repository:
+
+```
+git clone https://github.com/exasol/tableau-connector.git
+```
+
+* Package the connector with the script we provide (requires python installed):
+
+```
+/bin/bash <path to connector directory>/tableau-connector/tableau-server-GUI-tests/set_up_scripts/package_connector.sh
```

Comment:
that won't work on windows. I guess that most percent of the tableau users use windows, so we should maybe also add a guide for them.

Reply:
Or at least mention that you need a linux for those steps

Reply:
This is the command I used on Windows actually. I've mentioned that I use Git bash, maybe I can add a note here again about it